lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -71,22 +71,36 @@ export abstract class PerformanceClient implements IPerformanceClient {
     }
 
     /**
-     * Starts and returns an platform-specific implementation of IPerformanceMeasurement.
+     * Generates and returns a unique id, typically a guid.
```

Comment:
No changes to `generateId()`. Moving so abstract implementations are at the top.